cell.hoc

```diff
@@ -69,6 +69,7 @@ proc biophys() {
     insert pas
       g_pas = 0.001
       e_pas = -65
+    insert hh
```

Comment:
TODO: check if there are any reference files in the repository that need to be updated because of this change.

Reply:
Done in a4e834a0e943a64120269ce36222be5c37c45bd4.